src/main/java/com/okta/sdk/clients/LogsApiClient.java

```diff
@@ -0,0 +1,285 @@
+package com.okta.sdk.clients;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.google.common.annotations.VisibleForTesting;
+import com.okta.sdk.framework.ApiClientConfiguration;
+import com.okta.sdk.framework.ApiResponse;
+import com.okta.sdk.framework.FilterBuilder;
+import com.okta.sdk.framework.JsonApiClient;
+import com.okta.sdk.framework.PagedResults;
+import com.okta.sdk.framework.Utils;
+import com.okta.sdk.models.log.Log;
+import org.apache.http.HttpResponse;
+import org.joda.time.DateTime;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LogsApiClient extends JsonApiClient {
+
+    private static final String START_DATE = "since";
+    private static final int DEFAULT_RESULTS_LIMIT = 100;
+
+
+    public LogsApiClient(ApiClientConfiguration config) {
+        super(config);
+    }
+
+    ////////////////////////////////////////////
+    // COMMON METHODS
+    ////////////////////////////////////////////
+
+    /**
+     *
```

Comment:
because this is an sdk, can you please add javadoc to describe what the method does?